README.md

```diff
@@ -1,634 +1,1855 @@
-
-
-##### Features that Cardano-Loans shares with other *distributed dApp* protocols:
-
-- **Full Custody** - users always maintain full spending *and* delegation control over their assets.
-- **Natural Concurrency** - throughput scales *with* the number of users. No specialized batchers/indexers are necessary, though they *may* be used in proprietary backends of large lending providers.
-- **Frontend/Backend Agnosticism & Interoperability** - relatively straightforward integration with existing frontends (i.e. wallets). Third-party platforms can build business models atop open-contract p2p protocols. 
-- **Censorship Resistance** - due to the aforementioned agnosticism, distributed p2p dApps offer the highest level of censorship-resistance of any dApp architecture.
-- **No Superfluous "dApp" Tokens** - ADA is all you need to pay for script/TX fees. 
-- **Democratic Upgradability** - users choose if/when to use new contracts.
+## Motivation
 
+A healthy credit-debt market is a vital (and often the largest) component of a thriving economy. A
+healthy market is one in which prices reflect underlying reality as quickly and faithfully (with as
+little friction) as possible. For the sake of avoiding a whole treatise on economics, suffice to say
+that *the best way to achieve fast and frictionless price discovery is through the aggregation of
+maximally expressive individual sentiments.* This is especially true for credit-debt markets, where
+money itself is the asset, priced via interest rates. Futhermore, as with all DeFi protocols, any
+credit-debt market that seeks to be an alternative to the current global financial system will
+likely be perceived as a threat by those in positions of high power, so censorship resistance is an
+essential feature. 
+
+Unfortunately, contemporary lending/borrowering DApps on Cardano are implemented in ways that cannot
+possibly create a true trustless and decentralized credit-debt market. To fully appreciate this, it
+is first important to understand the deficiencies of the status quo:
+
+### Oracles
+
+DApps that rely on off-chain information feeds are subject to the integrity of the underlying oracle
+network. It is pretty easy to find examples of faulty oracle feeds causing DeFi issues that resulted
+in users losing money. Not only that, but in many cases, *oracles are simply not necessary*. As long
+as users are able to negotiate the terms *before* initiating the loan, there is no need for
+liquidations, and therefore, no need for oracles. In TradFi, liquidations are usually only used
+with *margin loans* which are typically short-term loans and/or are generally only used to buy more
+securities (ie, speculate on markets). In a typical economy, most loans are *not* margin loans; you
+wouldn't use a margin loan to start a business.
```

Comment:
I like your framing for why "cardano-loans" here and I don't want you to change the document because this is obviously the philosophy of cardano-loans.

However, I don't agree.  I think that all loans are heading in the direction of being treated as margin loans, and this is particularly so since the 2008 GFC.  For example, even the current value of US treasuries held by banks, guaranteed to redeem at full value, had to be manipulated by the FED with their recent BTFP program.  IE: Even those "guaranteed by Govt" assets caused bank liabilities to act like margin loans.  I think we are gradually heading into a new world where every asset is valued constantly, even houses when they become tokenised and fractionally saleable.

Reply:
>  I think we are gradually heading into a new world where every asset is valued constantly, even houses when they become tokenised and fractionally saleable.

I agree that society is trending this way, but I am not convinced it will be successful. I think there are physical limits to how continuous real-time price discovery can be, especially on blockchain. For example, network bandwidth probably cannot support all of the traffic required to support real-time price discovery of everything; every time someone updates a price, the network needs to transmit that difference to everyone. Day-traders can accidentally ddos the network. 

Then, you have the blockchain bandwidth of constantly updating the state on-chain using some kind of oracle. Assuming there is actually a good solution to oracles, can they update the prices fast enough to keep up while still allowing network space for other activities? I personally wouldn't bet on this.